1.9/administration/metrics/metrics-api.md

```diff
@@ -3,6 +3,30 @@ post_title: Metrics API
 menu_order: 1
 ---
 
+You can use your custom built tools to get any metric, either from running Mesos tasks or the hosts which run my DC/OS cluster. 
+
+The HTTP producer is enabled by default and exposes a JSON-formatted HTTP API on each node in the cluster. These APIs include both metrics datapoints as well as dimensions, or key/value pairs with relevant node and cluster metadata.
+
+## Base Structure
+
+```
+http://<hostname>:<port>/system/v<version>/metrics/v<version>/<request>
+```
+
+## Node Endpoints
+
+```
+/system/metrics/v0/node
+```
+
+## Containers and App Endpoints
```

Comment:
Yes, I am adding in my next commit. The PR to support this on dcos-website was just merged: https://github.com/dcos/dcos-website/pull/1008